docs/user-guides/sbs_swarm_interface.md

```diff
@@ -0,0 +1,477 @@
+---
+title: "Step-by-Step: Swarm Interface"
+page_id: sbs_swarm_interface
+---
+
+Here we will go through step-by-step how to interface with a swarm of crazyflies and make all the copters of the swarm hover and fly simultaneously in a square shape using the `Swarm()` class of the cflib.For this tutorial you will need a swarm (2 or more) of crazyflies with the latest firmware version installed and a positioning system (Flowdeck,Lighthouse,LPS etc.) that is able to provide data for the position estimation of the crazyflies.
```

Comment:
Should we be stricter and limit it to absolute positioning systems, that is remove the flow?
It will probably work with flow decks but it adds one more layer of complexity with relative positioning and accumulated errors.

Reply:
Clarified better in changes